### ISSUE TITLE
Place q into builtins?

### ISSUE THREAD

**Issue by proofit404** [zestyping/q#50]
It is annoying to type `import` each time I want to use `q`.

Installation process can setup init hook so `q` or `_q` will be available in the global namespace.

Is it desired feature?

**Comment by naiveai** [comment]
Yeah, I personally think this is the main (and pretty much only) disadvantage of q - having to install it each time if you're using virtualenvs, for example. And remembering to uninstall it after you debug or it becomes part of your dependencies.

**Comment by merwok** [comment]
That’s a different thing!

The OP requested a way to have the q module automatically imported, the second message is about automatically installing the q package (which is a problem solved by virtualenvwrapper hooks or dev dependencies etc)
